patches/@edx+frontend-build+13.0.14.patch

```diff
@@ -0,0 +1,91 @@
+diff --git a/node_modules/@edx/frontend-build/config/jest.config.js b/node_modules/@edx/frontend-build/config/jest.config.js
+index ae08b70..952ccc3 100644
+--- a/node_modules/@edx/frontend-build/config/jest.config.js
++++ b/node_modules/@edx/frontend-build/config/jest.config.js
+@@ -3,12 +3,12 @@ const fs = require('fs');
+ 
+ const presets = require('../lib/presets');
+ 
+-let envConfigPath = path.resolve(__dirname, './jest/fallback.env.config.js');
+-const appEnvConfigPath = path.resolve(process.cwd(), './env.config.js');
++const appEnvConfigPathJs = path.resolve(process.cwd(), './env.config.js');
++const appEnvConfigPathJsx = path.resolve(process.cwd(), './env.config.jsx');
+ 
+-if (fs.existsSync(appEnvConfigPath)) {
+-  envConfigPath = appEnvConfigPath;
+-}
++const envConfigPath = fs.existsSync(appEnvConfigPathJs) ? appEnvConfigPathJs
++                      : fs.existsSync(appEnvConfigPathJsx) ? appEnvConfigPathJsx
++                      : path.resolve(__dirname, './jest/fallback.env.config.js');
```

Comment:
[sanity check] Will this `__dirname` will refer to the appropriate path when used in a `.patch`? Presumably the patch is brought into the `node_modules` during the `postinstall` so it should be OK; just wanted to see whether that's your understanding as well.